about.html

```diff
@@ -10,55 +10,91 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
+\
+
+<div class="container">   
+      <header class= "nav-bar">
+       <nav> 
+       <img class="logo" src="img/lambda-black.png" alt="Great Idea! Company logo.">
+           <a href="index.html">Services</a>
```

Comment:
Great job making it easier on yourself and moving the image into the nav here (another cool trick is you can wrap the `<img>` in a `<a>` and link it to the home page).